crates/forge/tests/it/revive/cheats_common.rs

```diff
@@ -0,0 +1,28 @@
+//! Individual cheatcode tests for pallet-revive
+//! Each test runs a specific cheatcode file for easier debugging
```

Comment:
One thing I noticed when I tried adding a lot of individual tests was that it was kind of overcoming our test machines, because they started executing in parallel